### PR TITLE
403 on projects.view on permission fail on projects

### PR DESCRIPTION
**Pull request by balendurawat**
Targets #1117

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by GauravGusain98** [comment]
Please fix the failed checks.

File: Modules/Project/Http/Controllers/ProjectController.php [hunk 1]
```diff
@@ -21,9 +21,13 @@ public function __construct(ProjectServiceContract $service)
...
     {
-        $projects = $this->service->index();
+        if (auth()->user()->hasPermissionTo('projects.view')) {
+            $projects = $this->service->index();
 
-        return view('project::index')->with('projects', $projects);
+            return view('project::index')->with('projects', $projects);
+        } else {
+            return view('errors/403');
+        }
```
Comment (GauravGusain98) [inline Modules/Project/Http/Controllers/ProjectController.php:30]: This way is not recommended to solve this problem. Please try other ways like middlewares or policies.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments.

**Comment by deepankar-sajwan** [comment]
Pull request is closed as code is already merged through diferent code request

**Pull request closed without merging**